site/_posts/2017-03-07-deletions-and-dragons.html.md

```diff
@@ -0,0 +1,63 @@
+
+There were approaching footsteps. Loud ones. He hastily and groggily rolled out of his bedroll and stood with nontrivial effort to stare blearily at the other end of the bridge. Striking a match, he peered into the dark. He could nigh make out the outline of the pesky invader who called out to him. "[Janmorel][jc] here. I am a Half-Orc Loremaster from Celene. Here to lend my assistance." Janmorel strode across to meet Jamlamin, who quickly realized that at the side of the Half-Orc was a massive war dog with a ridiculous red-sequined collar full of bells. "Look," Janmorel said. "I'm here to get rid of the beast, I really must. It has been dogging me for days now. I also have a [firmware migration][#14025] that you all might be interested in. As you can tell, the migration doesn't quite fit into Operating System, but plenty of your hardware types require such metadata and I'm here to provide the migration to do it!" The dog, apparently intent on ignoring everything but chasing leaves on the ground, had short black fur. Janmorel grimaced, dropped a strange looking bundle at the feet of Jamlamin, clicked his heels together, and vanished abruptly into the night. Jamlamin took one look at the dog and sighed heavily. "Maybe you'll make a decent guard after all. We do need one, you know." He bent to pick up the bundle, realizing as he did so that it had a strange locking mechanism that he doubted the party had the key to. 
+
+He locked the bundle in the company chest and realized to his chagrin that the dog had usurped most of his bedroll. The rest of the night passed uneventfully. In the morning, [Krisnys Bearcharger][kb], the female Elf Shadowdancer, had awoken before everyone else to make breakfast. [Nerisstina Cupshigh][nc], the female Gnome Paladin, growled at the dog who was eagerly sniffing for food, but reluctantly handed over half of her portion. [Eillyassa Chorster][ec], the female Gnome Ranger, was the last to awaken, partially because she was depressed. She had been jinxed in week four, and had the unfortunate burden of having a coatrack follow her around now. None of the company had yet been able to dispel the jinx, but after the hearty breakfast of what could only charitably be called dogfood, [Jamlamin Tarmikos][jt] stirred once again in Eillyassa's direction to attempt to sever the bond between her and the unfortunate piece of furniture. After a few moments of fumbling with the coatrack, Jamlamin straightened. "I think that the spell backfired. It was not specifying the correct collection. Try putting some distance between you and it, again, please?" The company watched in relief as Eillyassa left the coatrack in the dust. [Magnys Gladomain][mh], the female Elf Rogue, stammered. "Wh...what did you do?" Jamlamin replied, "I believe the [referencing objects via id on services create][#14087] was broken. Should work fine now." 
+
```

Comment:
`In the morning, [Krisnys Bearcharger][kb], the female Elf Shadowdancer, had awoken before everyone else to make breakfast.`

I don't know, this makes me 😆  uncontrollably since many a early morning and late evening I see github mentions from @kbrock.  I sometimes wonder what timezone he's in.  😉